site/content/docs/browser-checks/_index.md

```diff
@@ -145,6 +145,16 @@ We have picked a selection of handy templates that have been optimised for Playw
 
 ![checkly-browser-check-templates](/docs/images/browser-checks/browser-check-templates.png)
 
+### Editor tips
+
+The Checkly UI supports the following keyboard shortcuts within the browser check editor to perform routine actions directly from your keyboard.
```

Comment:
I'd go with a more personal tone, i.e.:  You can use the following keyboard shortcuts to perform routine actions within the browser check editor.